src/PresentationalComponents/RuleDetails/RuleDetails.js

```diff
@@ -93,48 +84,105 @@ const RuleDetails = ({ children, rule, intl, topics, header, isDetailsPage }) =>
                     <Stack>
                         <StackItem>{intl.formatMessage(messages.totalRisk)}</StackItem>
                         <StackItem className='pf-u-display-inline-flex alignCenterOverride pf-u-pb-sm pf-u-pt-sm'>
-                            <span className='pf-u-display-inline-flex'>
-                                <Battery
-                                    label=''
-                                    severity={rule.total_risk} />
-                                <span className={`batteryTextMarginOverride pf-u-pl-sm ins-sev-clr-${rule.total_risk}`}>
-                                    {AppConstants.TOTAL_RISK_LABEL[rule.total_risk] || intl.formatMessage(messages.undefined)}
-                                </span>
-                            </span>
-                        </StackItem>
-                        <StackItem>
-                            <TextContent>
-                                <Text component={TextVariants.small}>{intl.formatMessage(messages.rulesDetailsTotalriskBody, {
-                                    likelihood: AppConstants.LIKELIHOOD_LABEL[rule.likelihood] || intl.formatMessage(messages.undefined),
-                                    impact: AppConstants.IMPACT_LABEL[rule.impact.impact] || intl.formatMessage(messages.undefined),
-                                    strong(str) { return <strong>{str}</strong>; }
-                                })}</Text>
-                            </TextContent>
+                            <Split hasGutter>
+                                <SplitItem>
+                                    <InsightsLabel value={rule.total_risk}/>
+                                </SplitItem>
+                                <SplitItem isFilled/>
```

Comment:
hasGutter introduces enough space here... less this is specifically requested, don't think we need this

Reply:
So for this and the below, the mocks seem to want the right side of the top and bottom sections to be aligned. If I take out the extra `<SplitItem isFilled/>` they will appear at different distances. If that's alright then I can take it out 😁